ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino

```diff
@@ -826,35 +825,37 @@ void ReceiveResponses() {
         commandAcknowledged();
       }
       else if (parseTemperatures(serialResponse)) {
-        telnetSend("< AutoReportTemps parsed");
+        telnetSend("AutoReportTemps parsed");
         if (lastCommandSent.startsWith("M109") || lastCommandSent.startsWith("M190"))
           restartSerialTimeout();   // When firmware doesn't have 'BUSY_WHILE_HEATING' temperature sent during heating may be used to prevent timeout
         GotValidResponse();   // Warning, this will empty 'serialResponse'
       }
       else if (parsePosition(serialResponse)) {
-        telnetSend("< MPosition parsed");
+        telnetSend("MPosition parsed");
         if (lastCommandSent.startsWith("G28"))
           restartSerialTimeout();   // Some firmware doesn't send busy while homing but just position. It can be used to prevent timeout
         GotValidResponse();   // Warning, this will empty 'serialResponse'
       }
       else if (serialResponse.startsWith("echo:busy")) {
```

Comment:
Which firmware sends this? Sure there is no blank after ":"?
Maybe it would be more robust to check if the line contains the string "busy"?